src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="TESTHOST/testfqdn.test.corefx.net"
+principal2="TESTHTTP"
+krb_user="krb_user"
+password="password"
```

Comment:
I know steps have been taken to mitigate concerns related to this, but are there additional ones we can take?  For example, do these credentials need to be hard coded, or could the setup script generate them randomly?